packages/twenty-front/src/modules/object-record/record-field/components/FieldInput.tsx

```diff
@@ -129,6 +131,8 @@ export const FieldInput = ({
         <RatingFieldInput onSubmit={onSubmit} />
       ) : isFieldSelect(fieldDefinition) ? (
         <SelectFieldInput onSubmit={onSubmit} onCancel={onCancel} />
+      ) : isFieldMultiSelect(fieldDefinition) ? (
+        <MultiSelectFieldInput onCancel={onCancel} />
```

Comment:
We should have a onSubmit here too

Reply:
Not sure, onSubmit closes the multi-select edit input which is not a good user experience. I just call the `persistField` when using the edit input